src/midgard/polyline2.cc

```diff
@@ -64,7 +64,7 @@ void Polyline2<coord_t>::Generalize(container_t& polyline,
   peucker = [&peucker, &polyline, epsilon, &indices](typename container_t::iterator start, size_t s,
                                                      typename container_t::iterator end, size_t e) {
     // find the point furthest from the line
-    float dmax = std::numeric_limits<typename coord_t::value_type>::lowest();
+    double dmax = std::numeric_limits<typename coord_t::value_type>::lowest();
```

Comment:
This should probably be `coord_t::value_type` instead of `double`, or `auto` to avoid this happening again when we move to `bigdouble` :-P

Reply:
👍  Fixed